tests/compute/subresources/reset.go

```diff
@@ -47,31 +47,20 @@ var _ = compute.SIGDescribe("Reset subresource", func() {
 			By("Checking that the VirtualMachineInstance console has expected output")
 			Expect(console.LoginToAlpine(vmi)).To(Succeed())
 
-			errChan := make(chan error)
```

Comment:
Would you consider a simpler test:
* log into the guest
* touch /run/non-persistent-file
* reset
* log into guest
* check that the file is gone

Reply:
/lgtm

Thanks !

I think that the current version would succeed if we replaced Reset with Restart. To be holier, you can watch the vmi and expect it to never move out of `Ready`.

Reply:
>To be holier, you can watch the vmi and expect it to never move out of Ready.

But this will not guarantee that the reset was performed, or am I missing something?

Reply:
Maybe I was not clear. I suggested to *add* a VMI watch. Seeing that the file is gone tells us that reset (or restart) took place. Seeing that it's the same VMI object tells us that it was a guest-internal reset.

Reply:
Got it, thanks. I opted for checking the vmi UUID is still the same.